jadx-plugins/jadx-rename-mappings/src/main/java/jadx/plugins/mappings/save/MappingExporter.java

```diff
@@ -144,10 +144,12 @@ public void exportMappings(Path path, JadxCodeData codeData, MappingFormat mappi
 
 			String srcNamespace = MappingUtil.NS_SOURCE_FALLBACK;
 			String dstNamespace = MappingUtil.NS_TARGET_FALLBACK;
+
+			// Copy mappings from potentially imported mappings file
 			if (loadedMappingTree != null && loadedMappingTree.getDstNamespaces() != null) {
-				srcNamespace = loadedMappingTree.getSrcNamespace();
-				dstNamespace = loadedMappingTree.getDstNamespaces().get(0);
```

Comment:
@NebelNidas are you sure these namespaces assigns should be removed?

Reply:
`accept` visits the namespaces automatically, so it should be fine: https://github.com/FabricMC/mapping-io/blob/50e0877d700729dc29828713629f9e086a0f18c2/src/main/java/net/fabricmc/mappingio/tree/MemoryMappingTree.java#L216-L219